frontend/src/components/search/SearchErrorBoundary.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {ErrorBoundary} from "@appbaseio/reactivesearch";
+import {V2} from "../../openapi";
+import {Navigate} from "react-router-dom";
+
+
+import Cookies from "universal-cookie";
+
+const cookies = new Cookies();
+
+export function SearchErrorBoundary(props) {
+
+	const {children} = props;
+
+	return (
+		<ErrorBoundary
+			renderError={error => (
+				<>
+					{
+						(() => {
+							if (error["status"] === 401 || error["status"] === 403) {
+								V2.OpenAPI.TOKEN = undefined;
+								cookies.remove("Authorization", {path: "/"});
+								return <Navigate to="/auth/login"/>;
```

Comment:
I cannot properly do the refresh token logic here; for now just redirect to the login page